demo/EXPLOR3R/auto-drive.py

```diff
@@ -48,6 +48,8 @@
 ir = InfraredSensor(); assert ir.connected
 ts = TouchSensor();    assert ts.connected
 
+print('Robot Starting')
```

Comment:
There could be something about it here:
https://github.com/rhempel/ev3dev-lang-python#user-resources.